### PR TITLE
Update to kotlin 1.7.10

### PR DESCRIPTION
**Pull request by saket**
I ran all the samples and they look fine.

### CODE REVIEW

File: mosaic/build.gradle [hunk 1]
```diff
@@ -60,10 +63,9 @@ allprojects {
 		}
 	}
 
-	tasks.withType(org.jetbrains.kotlin.gradle.dsl.KotlinJvmCompile).configureEach { task ->
+	tasks.withType(KotlinJvmCompile).configureEach { task ->
```
Comment (saket) [inline mosaic/build.gradle:66]: IntelliJ was showing yellow lines for these fully qualified names. I can undo these imports if you want.

**Comment by JakeWharton** [comment]
We can actually nuke the submodule and switch to JetBrains' runtime like we do in Molecule and Redwood! I'll do in a follow-up.

**Comment by saket** [comment]
that'd be awesome!

**Comment by JakeWharton** [comment]
Tracked by #55

**Pull request merged**